ccmlib/node.py

```diff
@@ -974,20 +974,19 @@ def clear(self, clear_all=False, only_data=False, saved_caches=False):
         self._create_directory()
 
     def run_sstable2json(self, out_file=None, keyspace=None, datafiles=None, column_families=None, enumerate_keys=False):
-        print("running")
         if out_file is None:
-            out_file = sys.stdout
+            out_file = subprocess.PIPE
         sstable2json = self._find_cmd('sstabledump')
         env = self.get_env()
         sstablefiles = self.__gather_sstables(datafiles, keyspace, column_families)
-        print(sstablefiles)
+        common.print_if_standalone(str(sstablefiles), debug_callback=self.info, end='')
```

Comment:
could you remind me where i can find the definition of `self.info`. i searched for the assignment statement of this member variable, but no luck.